WebExtension/bridge.js

```diff
@@ -90,6 +91,16 @@ function init_bridge() {
 				});
 				return;
 			}
+			if (typeof(chrome) === 'undefined') {
+				if (!items.isProperlyMigrated) {
+					XKit.window.show("Storage migration in progress", "XKit is still busy migrating your preferences from the old storage system to the new one. Please check back in a couple seconds by refreshing the page.", "warning",  "<div class=\"xkit-button default\" id=\"xkit-bridge-refresh\">Refresh</div>");
```

Comment:
This doesn't look much prettier wrapped...what do I need to bribe you with? (alternatively, a few lines above there's an even worse offender)